### PR TITLE
Update ask-chatgpt extension

### PR DESCRIPTION
**Pull request by Ex10si0n**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/ask-chatgpt/package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  "icon": "command-icon.png",
+  "author": "Ex10si0n",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "chat",
+      "title": "Chat",
```
Comment (pernielsentikaer) [inline extensions/ask-chatgpt/package.json:15]:
```suggestion
      "title": "Ask ChatGPT",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

Could you add [metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) too

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by Ex10si0n** [comment]
I have fixed the `package.json` and added standard screenshot inside folder `metadata`, could you please have one more time to check it for me?

File: extensions/ask-chatgpt/package.json (hunk 1 above)
Comment (Ex10si0n) [reply extensions/ask-chatgpt/package.json]: well noted, I have changed

**Comment by pernielsentikaer** [comment]
Maybe we should make a collab with #5157 since it kinda does the same job so we don't end up with a lot of extensions doing the same - would you be up for that?

**Pull request closed without merging**